### ISSUE TITLE
text sometimes selects when clicking back into VSCode window

### ISSUE THREAD

**Issue by af4jm** [microsoft/vscode#130147]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.58.2 (but has been happening for a LONG time)
- OS Version: Win10

Steps to Reproduce:

1. Click out of VSCode to a browser (I use Brave, but a co-worker who doesn't have Brave sees the same thing)
2. click around, do whatever (for me it's usually select some text & copy to the clipboard, or scroll the mouse wheel to keep reading)
3. click back into VSCode

Expected behavior: VSCode comes to the foreground, cursor is at wherever I clicked

Actual Behavior sometimes: the text is selected from where the cursor was when I clicked off VSCode to where it moved to when I clicked back in, making it very easy to type or paste over code when the intention is to keep that and add to it

I say sometimes because it doesn't happen all the time & I can't figure out a pattern, it's often enough to be really annoying though

**Comment by hediet** [comment]
Just to make sure I fully understand the issue, can you share a gif/video?

Is this right:
* the cursor is at position p1
* you switch to your browser (does this happen with any other app too?)
* you switch back to VS Code by clicking on position p2

Expected outcome: Cursor is at position p2
Actual outcome: Cursor is at position p2, but there is also a selection from p1 to p2.

**Comment by af4jm** [comment]
yes, this is correct... happens with multiple browsers on multiple PCs... but seemingly at random, I can't determine any pattern to it

**Comment by hediet** [comment]
Could you share a gif so we can see some context?
Is this specific to the Brave browser?

**Comment by af4jm** [comment]
apologies for the delayed response... it's not specific to Brave, as I've had it happen with the Kindle app as well... as for a gif recording, I can try, although it is kind of random as to when it happens or not (usually not)
